docs/public/sdk/GettingStarted.rst

```diff
@@ -1,4 +1,208 @@
 Getting started
----------------
+^^^^^^^^^^^^^^^
 
-TBD
+Opentelemetry C++ SDK provides the reference implementation of Opentelemetry C++ API, 
+and also provides implementation for Processor, Sampler, and core Exporters as per the 
+specification.
+
+
+Exporter
+^^^^^^^^
+
+An exporter is responsible for sending the telemetry data to a particular backend. 
+OpenTelemetry offers six tracing exporters out of the box:
+
+- In-Memory Exporter: keeps the data in memory, useful for debugging.
+- Jaeger Exporter: prepares and sends the collected telemetry data to a Jaeger backend via UDP and HTTP.
+- Zipkin Exporter: prepares and sends the collected telemetry data to a Zipkin backend via the Zipkin APIs.
+- Logging Exporter: saves the telemetry data into log streams
```

Comment:
nit: ending `.`? The same for line 20.

Reply:
thanks, t was missing at few other places, fixed all.